Cryptocurrency/bitso-prices.10s.sh

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env bash
+
+# <xbar.title>Bitso Price (USD / MXN)</xbar.title>
+# <xbar.version>v1.0</xbar.version>
+# <xbar.author>Alex Ventura</xbar.author>
+# <xbar.author.github>alexventuraio</xbar.author.github>
+# <xbar.desc>Displays BTC, LTC, ETH, XRP and BCH prices from Bitso either in USD or MXN</xbar.desc>
+# <xbar.image>https://i.ibb.co/ZRFdnxpC/Screenshot-2025-03-24-at-14-25-12.png</xbar.image>
+# <xbar.dependencies>curl, bash, jq</xbar.dependencies>
+
+# If you find this small script someway useful, please consider a tip or buying me a coffee:
+# Bitcoin: 3DUzWfDWR75AHuwnZcF7KyQAHLXf3iu7YV
+# Litecoin: MLjstS6a4a4AgXSd2cLhznPNjDhF5sucmr
+
+export LANG='en_US.UTF-8'
+export LC_NUMERIC='en_US'
+export PATH="/usr/local/bin/:$PATH"
+export PATH=/opt/homebrew/bin:$PATH
+
+# Show prices in USD or MXN
+SHOW_PRICES_IN_USD="true"
```

Comment:
This could likely be an [xbar variable](https://xbarapp.com/docs/2021/03/14/variables-in-xbar.html)